add_up.rb

```diff
@@ -0,0 +1,14 @@
+#Write a program with a function add_up(i) .  
+#It is to be passed a positive integer, and it will add all the numbers from 1 to that integer and return the sum.  
+#Call the function three times within the program, and each time print out the return value.  
+#Call the program add_up.rb.
+def add_up(i)
+    sum = 0
+   for item in 1..i do
+     sum = sum + item
+   end
+    return sum
+ end
```

Comment:
I thought the program was supposed to ask the user for the numbers, not being put manually in the code. This code is alright, but I understand the question is not very clear. I would do this way (totally optional).

![Screenshot from 2020-08-28 11-24-47](https://user-images.githubusercontent.com/47392623/91604675-f6163880-e923-11ea-9cba-b3fe3a2041d1.png)